captum/attr/_utils/summarizer.py

```diff
@@ -145,24 +165,23 @@ def CommonSummarizer():
     Returns a summarizer with common summary statistics, specifically with:
         Mean, Sample Variance, Sample Std Dev, Min, Max
     """
-    from captum.attr._utils.stat import Mean, Var, StdDev, Min, Max
-
     return Summarizer([Mean(), Var(order=1), StdDev(order=1), Min(), Max()])
 
 
 class SummarizerSingleTensor:
     r"""
         A simple class that summarizes a single tensor. The basic functionality
         of this class is two operations .update and .summary
+    """
 
+    def __init__(self, stats: List[Stat], summary_stats_indices: List[int]):
+        r"""
         Args:
-            summary_stats (list of Stat): A list of Stat objects you
-                want to show in the .summary property.
             stats (list of Stat): A list of all the Stat objects that
```

Comment:
It might be good to clarify here that this list needs to already contain all dependencies in the appropriate order / after applying _reorder_stats. It also might be good to not recommend users try to access this class directly and avoid generating documentation for this class on the website.

Reply:
I'm not sure how to avoid generating documentation for this class, unless I don't document it.

Reply:
We'll have to add these classes to rst files in order for documentation to be generated, so just a suggestion for that. We'll do that in a separate PR, so don't worry about it for merging this.